hazelcast-oss/Dockerfile

```diff
@@ -21,6 +21,7 @@ ARG EUREKA_PLUGIN_URLS='https://repo1.maven.org/maven2/com/hazelcast/hazelcast-e
 
 # Runtime variables
 ENV HZ_HOME="${HZ_HOME}" \
+    HZ_VERSION="${HZ_VERSION}" \
```

Comment:
Could it be resolved without setting env? Isn't ARG enough to refer `hazelcast-all-${HZ_VERSION}.jar`?

Reply:
good point, removed 👍